xds/internal/balancer/edsbalancer/eds.go

```diff
@@ -196,6 +196,7 @@ func (x *edsBalancer) handleGRPCUpdate(update interface{}) {
 
 		x.config = cfg
 	case error:
+		x.logger.Warningf("Received error from resolver: %v", u)
```

Comment:
Move to `handleErrorFromUpdate`?  (Is that method called from other places? Do they log?)

Reply:
Done.